### PR TITLE
Update docs for setup full node and validator

### PR DESCRIPTION
**Pull request by kwunyeung**
## Description

Update docs with full node setup and remove some unconfirmed information for validator.

## Checklist
- [x] Targeted PR against correct branch.
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Wrote tests.
- [ ] Added an entry to the `CHANGELOG.md` file.
- [x] Re-reviewed `Files changed` in the Github PR explorer.

### CODE REVIEW

**Comment by RiccardoM** [comment]
@kwunyeung Currently on the `master` branch there's already the ["*Node setup*" page](https://github.com/desmos-labs/desmos/blob/master/docs/validators/node-setup.md) that I've copied from the Cosmos instructions and adapted to Desmos. 

I think we should delete that if we decide to keep your instructions instead.

File: docs/genesis.json [hunk 1]
```diff
@@ -0,0 +1,395 @@
+{
```
Comment (RiccardoM) [inline docs/genesis.json]: I would advise against putting the `genesis.json` file here and, instead, putting it into the [Morpheus repo](https://github.com/desmos-labs/morpheus) to keep everything more organized.

File: docs/validators/validator-setup.md [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
+
 [Validators](overview.md) are responsible for committing new blocks to the blockchain through voting. A validator's stake is slashed if they become unavailable or sign blocks at the same height. Please read about [Sentry Node Architecture](validator-faq.md#how-can-validators-protect-themselves-from-denial-of-service-attacks) to protect your node from DDOS attacks and to ensure high-availability.
 
 ::: danger Warning
 If you want to become a validator for the `mainnet`, you should [research security](security.md).
 :::
 
+## Running a Fullnode
+
+To become a validator, you must first have `desmosd` and `desmoscli` installed and be able to run a fullnode. You can first [setup your fullnode](/fullnode/overview.html) if you haven't yet.
```
Comment (RiccardoM) [inline docs/validators/validator-setup.md]: I think it's better to change the link to the MD file to make it easier to navigate to even from the repo and IDEs. 

```suggestion
To become a validator, you must first have `desmosd` and `desmoscli` installed and be able to run a fullnode. You can first [setup your fullnode](/fullnode/overview.md) if you haven't yet.
```

**Comment by kwunyeung** [comment]
> I think we should delete that if we decide to keep your instructions instead.

I'm reading the original node setup and see what should be kept and linked.

File: docs/genesis.json (hunk 1 above)
Comment (kwunyeung) [reply docs/genesis.json]: updated the url

**Comment by kwunyeung** [comment]
I have linked the node installation to the full node installation I wrote. Please see if this works.

**Pull request merged**